earthly/cspell/Earthfile

```diff
@@ -16,8 +16,8 @@ CHECK:
     WORKDIR /work
 
     COPY $src .
-    
-    RUN cspell-cli lint . --dot
+
+    RUN cspell-cli --quiet --fail-fast lint . --dot
```

Comment:
I would remove `--fail-fast` so we can discover all the linting errors in the first run. What do you think?

Reply:
Yea we definitely dont want a `--fail-fast` and collect all issues at ones.

Reply:
Makes sense, done